### PR TITLE
System tests: rename tolerance_is_reller to tolerance_is_rel_err

### PR DESCRIPTION
**Pull request by soininen**
This PR renames the `tolerance_is_reller` flag in `MantidSystemTest` class to `tolerance_is_rel_err` which is in line with `CompareWorkspaces`. Also, a note about the absoluteness/relativeness of tolerances in system tests has been added to the dev-docs.

**Report to:** [nobody].

**To test:**

Build the `dev-docs` target and check that the System Tests page looks OK.

*There is no associated issue.*

*This does not require release notes* because this is an internal developer facing change.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Testing/SystemTests/lib/systemtests/systemtesting.py [hunk 1]
```diff
@@ -303,7 +303,7 @@ def validateWorkspaces(self, valNames=None, mismatchName=None):
         checker.setPropertyValue("Workspace1", valNames[0])
         checker.setPropertyValue("Workspace2", valNames[1])
         checker.setProperty("Tolerance", float(self.tolerance))
-        if hasattr(self, 'tolerance_is_reller') and self.tolerance_is_reller:
+        if hasattr(self, 'tolerance_is_relerr') and self.tolerance_is_relerr:
```
Comment (peterfpeterson) [inline Testing/SystemTests/lib/systemtests/systemtesting.py]: Since you are changing it anyhow, what about `self.tolerance_is_rel_err`?

Comment (soininen) [reply Testing/SystemTests/lib/systemtests/systemtesting.py]: It would be even more in line with `CompareWorkspaces`, then. Will do.

**Review by peterfpeterson — APPROVED** [review]
I was starting to wonder what a "reller" was.

**Comment by martyngigg** [comment]
:+1: for the rename too.

**Pull request merged**